index.html

```diff
@@ -13125,7 +13135,7 @@ <h2>Interface Mixin <dfn>AriaAttributes</dfn></h2>
 				attribute FrozenArray&lt;Element&gt;? ariaControlsElements;
 				attribute DOMString? ariaCurrent;
 				attribute FrozenArray&lt;Element&gt;? ariaDescribedByElements;
-				attribute Element? ariaDetailsElement;
+				attribute FrozenArray? ariaDetailsElements;
```

Comment:
I believe this should be `FrozenArray<Element>` like the other FrozenArrays of Elements:
```suggestion
				attribute FrozenArray&lt;Element&gt;? ariaDetailsElements;
```

Reply:
Wow good catch!